### DISCUSSION TITLE
Archive Searching

### DISCUSSION THREAD

**Discussion by jdhitsolutions** [buttondown/discussions#22] [Q&A] [answered]
Is there any option for searching archives? When I look at https://buttondown.email/behind-the-powershell-pipeline/archive/ I don't see any way to search by back catalog of content. And on a related topic, is archived content indexed by search engines, including paid content?

**Comment by jmduke** [comment, marked as answer]
Re: searching archives — not at the moment! It's a fairly popular request that we will probably get to _eventually_, but I tend to be a little hesitant building out explicitly _blog_-y features in the archives, since it's a never-ending pit of work :) 

Re: indexing — yes! No need for you to do anything; this happens automatically.
